src/ert/_c_wrappers/enkf/lark_parser.py

```diff
@@ -366,28 +396,19 @@ def get_values(item: SchemaItem, line: List[FileContextToken]):
     def define(self, tree: Tree):
         kw = tree.children[0]
         value = tree.children[1].children[0]
-        if not isinstance(value, FileContextToken):
-            raise ConfigValidationError(
-                f"Cannot define {kw!r} to {value!r}", config_file=self.config_file
-            )
+        assert_is_token(value, kw)
         self.defines.append([kw, substitute(self.defines, value)])
 
     def data_kw(self, tree: Tree):
         kw = tree.children[0]
         value = tree.children[1].children[0]
-        if not isinstance(value, FileContextToken):
-            raise ConfigValidationError(
-                f"Cannot define {kw!r} to {value!r}", config_file=self.config_file
-            )
+        assert_is_token(value, kw)
         self.data_kws.append([kw, substitute(self.defines, value)])
 
     def keyword(self, tree: Tree):
         arguments = []
         kw = tree.children[0]
-        if not isinstance(kw, FileContextToken):
-            raise ConfigValidationError(
-                f"Unrecognized keyword {kw!r}", config_file=self.config_file
-            )
+        assert_is_token(kw, kw)
```

Comment:
This is going raise and exception while raising an exception

Reply:
I heard you like exceptions...